src/ctapipe/tools/calculate_pixel_stats.py

```diff
@@ -68,16 +56,40 @@ class PixelStatisticsCalculatorTool(Tool):
 
     overwrite = Bool(help="Overwrite output file if it exists").tag(config=True)
 
+    append = Bool(help="Append to output file if it exists").tag(config=True)
```

Comment:
This should be an alias to `HDF5Merger.append`, not it's own option

Reply:
I had some problems with the append flag in the calibpipe CI. Simply adding "`--append`" to the pix stats tool was somehow not giving the desired results.